### ISSUE TITLE
juniper_hyper example broken

### ISSUE THREAD

**Issue by TomasBouska** [graphql-rust/juniper#270]
Hello,

I can't get juniper/juniper_hyper/examples/hyper_server.rs compiled due to 4th parameter introduction into juniper_hyper::graphql by commit 2e5df9f8a469943d694a1deb67910cbbdcd6cae4.

I realize this is a trivial fix, but I'm totally new to Rust, so I can't figure it out.

Thank you.

Cheers, Tomáš Bouška

**Comment by LegNeato** [comment]
Thanks for the report! I can't seem to reproduce this issue. Does it still persist after running `cargo update`?

**Comment by jatsrt** [comment]
The code that is no longer correct is:
`Box::new(juniper_hyper::graphql(root_node, ctx, req)),`

The latest version of juniper_hyper requires a new first parameter, a futures-pool.  

What is needed is:
`Box::new(juniper_hyper::graphql(pool, root_node, ctx, req))`

Need to do something like...
```
let pool = CpuPool::new(4);
    let addr = ([127, 0, 0, 1], 3000).into();

    let ctx = Arc::new(Context {
        client: DynamoDbClient::new(Region::UsWest2),
    });
    let root_node = Arc::new(Schema::new(Query, EmptyMutation::new()));

    let new_service = move || {
        let pool = pool.clone();
        let root_node = root_node.clone();
        let ctx = ctx.clone();
        service_fn(move |req| -> Box<Future<Item = _, Error = _> + Send> {
            let pool = pool.clone();
```

Though I'm new to rust too, so not sure if that is the right way to do it.

**Comment by TomasBouska** [comment]
> Thanks for the report! I can't seem to reproduce this issue. Does it still persist after running `cargo update`?

No, running `cargo update` didn't help. 

Four parameters are still required by juniper_hyper::graphql

**Comment by LegNeato** [comment]
Oh, I see what is going on here I think! The most recent released version on crates.io (0.1) uses 4 params (takes a  CPUPool from `futures_cpupool`). If you are using that version, you need to look at this example:

https://docs.rs/crate/juniper_hyper/0.1.0/source/examples/hyper_server.rs

On master, what will be released soon as 0.2 gets rid of needing a CpuPool and thus only takes three arguments. If you are using master / pointing to the git repo, you need to look at the example on master.

I plan to release 0.2 with the CpuPool removal this week.

**Comment by LegNeato** [comment]
Please let me know if that's the issue and I'll close!
